tests/cli/partition_test/multiple_partitions.bats

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bats
+
+TEST_HELPER_DIR="$BATS_TEST_DIRNAME/../test_helper"
+export TEST_HELPER_DIR
+
+load "$TEST_HELPER_DIR"/tools_check.bash
+load "$TEST_HELPER_DIR"/fluvio_dev.bash
+load "$TEST_HELPER_DIR"/bats-support/load.bash
+load "$TEST_HELPER_DIR"/bats-assert/load.bash
+
+setup_file() {
+    export FLV_LOG_BASE_DIR="${PWD}/flv-local"
+    export FLUVIO_PROFILE_LOCAL="partest-local"
+    export STREAMING_CONTROLLER_PUB_ADDR="127.0.0.1:9008"
+    export STREAMING_CONTROLLER_PRV_ADDR="127.0.0.1:9016"
+    export STREAMING_PROCESSING_UNIT_ONE_PUB_ADDR="127.0.0.1:9116"
+    export STREAMING_PROCESSING_UNIT_ONE_PRV_ADDR="127.0.0.1:9117"
+    export STREAMING_PROCESSING_UNIT_TWO_PUB_ADDR="127.0.0.1:9216"
+    export STREAMING_PROCESSING_UNIT_TWO_PRV_ADDR="127.0.0.1:9217"
+    export STREAMING_PROCESSING_UNIT_ONE_ID=1016
+    export STREAMING_PROCESSING_UNIT_TWO_ID=2016
+
+    PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME=$(random_string)
+    export PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME
+
+    PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME=$(random_string)
+    export PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME
+    debug_msg "Topic name: $PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME"
+
+    echo "Finishing Setup Process"
+    sleep 10
+}
+
+teardown_file() {
+    echo "Tearing down, shutting down cluster components"
+    "$FLUVIO_BIN" topic delete "$PRODUCE_CONSUME_MULTIPLE_PARTITIONS_TOPIC_NAME"
+    fluvio cluster shutdown --local
+}
+
+@test "Setup environment" {
```

Comment:
let's move out of this file since it's nor specific related to partitioning test